module/User/view/user/api-settings/create-principal.phtml

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+use Application\View\HelperTrait;
+use Laminas\Form\View\Helper\FormMultiCheckbox;
+use Laminas\View\Renderer\PhpRenderer;
+
+/** @var PhpRenderer|HelperTrait $this */
+
+$form->prepare();
+$form->setAttribute('action', $this->url('settings/api-principals/create'));
+$form->setAttribute('method', 'post');
+
+$form->setAttribute('role', 'form');
+$form->setAttribute('class', 'form-horizontal');
+?>
+<div class="row">
+    <h1><?= $this->translate('Create API token') ?></h1>
+    <?= $this->form()->openTag($form) ?>
+
+    <?php
+    $element = $form->get('description');
+    $element->setAttribute('class', 'form-control');
+    $element->setAttribute('placeholder', $element->getLabel());
+    ?>
+    <div class="form-group<?= count($element->getMessages()) > 0 ? ' has-error' : '' ?>">
```

Comment:
We should add a `BootstrapElementError` view helper a la [GEWIS/gewisweb](https://github.com/GEWIS/gewisweb/blob/master/module/Application/src/View/Helper/BootstrapElementError.php).

Reply:
f4042a579b66890875557fe9e266cb2f2944a6ec